h/data_tasks/report/create_from_scratch/08_annotation_counts/02_initial_fill.sql

```diff
@@ -0,0 +1,11 @@
+DROP INDEX IF EXISTS report.annotation_counts_created_week_authority_id_idx;
+
+REFRESH MATERIALIZED VIEW report.annotation_counts;
+
+ANALYSE report.annotation_counts;
+
+-- A unique index is mandatory for concurrent updates used in the refresh
+CREATE UNIQUE INDEX annotation_counts_created_week_authority_id_idx
+    ON report.annotation_counts (
+        created_week, authority_id, group_id, user_id, sub_type, shared
+    );
```

Comment:
Ok, I'm guessing `02_initial_fill.sql` gets run after `01_create_view.sql` above and somehow "fills" the empty view with data. I'm guessing `REFRESH MATERIALIZED VIEW` fills the view with data or maybe the view already has data in it and this updates it. And then we need to optimise things by `ANALYSE`'ing and creating an index